FastSurferCNN/run_prediction.py

```diff
@@ -325,28 +336,23 @@ def conform_and_save_orig(
 
         # Save input image to standard location, but only
         if subject.can_resolve_attribute("copy_orig_name"):
-            self.pool.submit(self.save_img, subject.copy_orig_name, orig_data, orig)
-
-        if not conf.is_conform(
-            orig,
-            conform_vox_size=self.vox_size,
-            check_dtype=True,
-            verbose=True,
-            conform_to_1mm_threshold=self.conform_to_1mm_threshold,
-        ):
-            LOGGER.info("Conforming image")
-            orig = conf.conform(
-                orig,
-                conform_vox_size=self.vox_size,
-                conform_to_1mm_threshold=self.conform_to_1mm_threshold,
-            )
+            self.async_save_img(subject.copy_orig_name, orig_data, orig, orig_data.dtype)
+
+        if not is_conform(orig, **self.__conform_kwargs(verbose=True)):
+            if (self.orientation is None or self.orientation == "native") and \
+                    not is_conform(orig, **self.__conform_kwargs(verbose=False, dtype=None)):
+                raise RuntimeError(
+                    f"To store images in native image space, the input image {subject.orig_name} must have isometric "
```

Comment:
```suggestion
                    f"To store images in native image space, the input image {subject.orig_name} must have isotropic "
```

Reply:
also check if you find other occurrences of "isometric".

Reply:
I will change that, but probably in a different PR. I have one lined up with only formatting changes. I am actually only testing that one as it is rebased on this PR.